argus/recipes/cloud/windows.py

```diff
@@ -36,7 +36,7 @@
 # Default values for an instance under booting step.
 COUNT = 20
 DELAY = 20
-_CBINIT_REPO = "https://github.com/openstack/cloudbase-init"
+_CBINIT_REPO = CONFIG.cbinit_repo
```

Comment:
Shouldn't this be `CONFIG.argus.cbinit_repo` ? Don't think it will work otherwise